src/main/pages/workspace-admin/installers.adoc

```diff
@@ -29,27 +31,36 @@ Installers are typically language servers and tools that provide features like s
 ----
 
 [id="how-installers-work"]
-== How Installers Work?
+== How installers work
+
+Installers are saved in a configuration file that a link:what-are-workspaces.html#bootstrapper[bootstrapper] uses to execute jobs. An installer script works exactly the same way as other shell scripts in Linux. The Che server checks if the launched process is running.
 
-Installers scripts are saved into a configuration file that a link:what-are-workspaces.html#bootstrapper[bootstrapper] uses to execute its jobs. An installer script is a shell script that works exactly the same way it’d work in your local Linux environment. Che server then checks if the process that an installer has launched is alive.
+Some installers activate special agents, such as the workspace, terminal, and exec agents.  If a workspace agent fails to start, the workspace is treated as if it has been started but the IDE is not usable. If the exec agent fails, the commands widget is unavailable.
 
-There are installers that activate special agents like workspace agent, terminal and exec agent. For example, if a workspace agent fails to start, the workspace will be considered as started but the IDE won’t be usable. Exec agent failure will result in unavailability of commands widget.
+[id="what-happens-when-enabling-and-disabling-installers"]
+== What happens when enabling and disabling installers 
 
-[id="enable-and-disable-installers"]
-== Enable and Disable Installers
+You can enable or disable installers per machine by using the *User Dashboard* or by updating the workspace machine configuration. When an installer is enabled, the bootstrapper executes an installer script after the workspace has started.
 
-Installers are can be enabled/disabled in User Dashboard per machine, or directly in workspace machine configuration. When an installer is enabled, bootstrapper executes an installer script after the workspace has started.
+The following shows installers that are enabled and disabled:
 
 image::workspaces/installers.png[]
 
-[id="installers-failures"]
-== Installers Failures
+[id="troubleshooting-installer-failures"]
+== Troubleshooting installer failures
+
+Installers run as if a user in the container may have `sudoers` privileges. If the user does not have privileges, the installer fails with `permission denied` issues.
```

Comment:
* headline the indiv. failures
* may have -> has
* -> does not have the privileges
* 'issues'? does it actually list some issues, or is it just an 'error message'?